diagnostic.md

```diff
@@ -94,4 +94,6 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+'excercise'
+'Jason' 'Weeks'
```

Comment:
teamMember['name'] would actually return the object that is store under the key of 'name': {given: 'Jason', surname: 'Weeks'}
Computers have no ability to interpret our prompts, so it would not be able to pick out only the values in that object without explicit direction to do so.